visualization/app/codeCharta/ui/metricChooser/metricValueHovered/metricValueHovered.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="metric-value" *ngIf="hoveredNode$ | async; let hoveredNode">
+	<div class="rounded-box">
+		<cc-metric-type [attributeType]="'nodes'" [metricFor]="metricFor"></cc-metric-type>
+		<span class="value">{{ hoveredNode.attributes[(primaryMetricNames$ | async)[metricFor]] }}</span>
+	</div>
+	<div
+		*ngIf="hoveredNode.deltas"
+		[style.marginLeft]="'2px'"
+		[style.backgroundColor]="
+			metricFor === 'heightMetric' && hoveredNode.deltas[(primaryMetricNames$ | async)[metricFor]] > 0
+				? '#b1d8a8'
+				: hoveredNode.deltas[(primaryMetricNames$ | async)[metricFor]] < 0
+				? '#ffcccc'
+				: '#e6e6e6'
```

Comment:
id love if those styles were to be applied by a class, rather than in the html code. Meaning giving this container conditional classes and setting the colors accordingly.

Reply:
as this are custom colors I am fine with the inline style. But I have no hard preferences

Reply:
Id love to hear how others think about this. My stance is that colors and everything that is style related should be handled by the scss. Is there a reason those are custom colors in the first place?